contents/docs/deployment/deploy-proxy.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Deploying a Reverse Proxy to Posthog Cloud 
+sidebar: Docs
+showTitle: true
+---
+<br>
+
+
+## Why host this reverse proxy? 
+
+You'll want to host this reverse proxy if you want to use Posthog Cloud but still want tracking data to be sent first to your domain, then to us. This helps with tracker blockers or if you have migrated or are migrating off Posthog open source hosted locally to Posthog Cloud.
+
+## Deploying the reverse proxy
+
+We like using Caddy because it makes setting up the reverse proxy and TLS a breeze.
+
+```bash
+docker run -p 80:80 -p 443:443 caddy caddy reverse-proxy --to p.posthog.com:443 --from <YOUR_TRACKING_DOMAIN>
```

Comment:
I'm a tiny bit concerned this is not production-ready. Should we just have an image for this? I also think we could add instructions on how to do this with AWS and/or Cloudflare. I could add them in a couple of days.

Reply:
This is totally production ready! The container would just be one more thing for us to manage. Caddy on a ti-83 would be able to serve all of the traffic we see on production. (almost)

FWIW you can't do this using AWS ELBs as they will only direct traffic internally.

Cloudflare proxying might work. I haven't tested that.

Reply:
Oh perfect, let's go then!

For the reverse proxy I actually meant something like a Cloudfront distribution, but we can keep this as an improvement.